tests/test_compute.py

```diff
@@ -297,6 +297,91 @@ def test_compute(client, publisher_wallet, consumer_wallet, free_c2d_env):
     assert result_data is not None, "We should have a result"
 
 
+@pytest.mark.integration
+def test_compute_arweave(client, publisher_wallet, consumer_wallet, free_c2d_env):
+    valid_until = get_future_valid_until()
+    ddo, tx_id, alg_ddo, alg_tx_id = build_and_send_ddo_with_compute_service(
+        client,
+        publisher_wallet,
+        consumer_wallet,
+        False,
+        asset_type="stored_in_arweave",
+        c2d_address=free_c2d_env["consumerAddress"],
+        valid_until=valid_until,
+        c2d_environment=free_c2d_env["id"],
+    )
+    sa_compute = get_first_service_by_type(alg_ddo, ServiceType.ACCESS)
+    sa = get_first_service_by_type(ddo, ServiceType.COMPUTE)
+    nonce, signature = get_compute_signature(client, consumer_wallet, ddo.did)
+
+    # Start the compute job
+    payload = {
+        "dataset": {"documentId": ddo.did, "serviceId": sa.id, "transferTxId": tx_id},
+        "algorithm": {
+            "serviceId": sa_compute.id,
+            "documentId": alg_ddo.did,
+            "transferTxId": alg_tx_id,
+        },
+        "signature": signature,
+        "nonce": nonce,
+        "consumerAddress": consumer_wallet.address,
+        "environment": free_c2d_env["id"],
+    }
+
+    # Start compute with valid signature
+    payload["signature"] = signature
+    response = post_to_compute(client, payload)
+    assert response.status == "200 OK", f"start compute job failed: {response.data}"
+
+    job_info = response.json[0]
+    print(f"got response from starting compute job: {job_info}")
+    job_id = job_info.get("jobId", "")
+
+    # get a new signature
+    nonce, signature = get_compute_signature(client, consumer_wallet, ddo.did)
+    payload = dict(
+        {
+            "signature": signature,
+            "nonce": nonce,
+            "documentId": ddo.did,
+            "consumerAddress": consumer_wallet.address,
+            "jobId": job_id,
+        }
+    )
+
+    compute_endpoint = BaseURLs.SERVICES_URL + "/compute"
```

Comment:
do you think we should test the result part? I think starting the compute job is enough and offers enough coverage to your work. If a file is downloadable, then it can be used in compute, so I wouldn't worry for a result.

Reply:
I'm okay with removing this result check.

Reply:
Done.